cmake/Modules/UseDoxygen.cmake

```diff
@@ -134,7 +159,9 @@ if(DOXYGEN_FOUND AND DOXYFILE_IN_FOUND)
 
 	configure_file("${DOXYFILE_IN}" "${DOXYFILE}" @ONLY)
 
-	get_target_property(DOC_TARGET doc TYPE)
```

Comment:
Hmm. Well, cmake not being Python, tabs are completely cosmetic... but no, I'd say if anything, the _rest_ of our files should be stripped of their shameful tabs. What year is this, 1995? :wink: 

```console
$ cd /usr/share/cmake/Modules

$ ls *.cmake|wc -l                                                    
360

$ grep -c '     ' *.cmake|egrep ':0$'|wc -l                           
337

$ for file in `grep -c '        ' *.cmake|egrep -v ':0$'|cut -d: -f1`; do echo "$(grep -c '     ' $file) $(wc -l $file)"; done|column --table --table-columns TABS,LINES,FILENAME --table-right TABS,LINES      
TABS  LINES  FILENAME
   5     38  DateTimeFormat.cmake
   1    389  FindLAPACK.cmake
  38    120  ManageAPIDoc.cmake
  95    341  ManageArchive.cmake
   1    188  ManageChangeLogScript.cmake
 105    244  ManageDependency.cmake
  28    168  ManageEnvironment.cmake
  17     84  ManageGConf.cmake
  92    251  ManageGettextScript.cmake
  19     99  ManageGitScript.cmake
   8     71  ManageMessage.cmake
  13     60  ManageRelease.cmake
  24    253  ManageReleaseFedora.cmake
 135    483  ManageRPM.cmake
 134    405  ManageRPMScript.cmake
  53    192  ManageSourceVersionControl.cmake
 127    450  ManageString.cmake
   9     59  ManageTarget.cmake
 233    669  ManageTranslation.cmake
  49    288  ManageUpload.cmake
 123    398  ManageVariable.cmake
 384    973  ManageZanata.cmake
  46    201  ManageZanataScript.cmake
```

337/360 files with no tabs, and the remaining few with a mishmash of tabbed lines and untabbed lines. (Having looked at some of those files, it's literally that: A file will be indented mostly with spaces, until you get 3/4 of the way down and then there's a block of lines that all start with a couple of tabs, and then indent the rest of the way with spaces.) 

...However, the file should be aligned _properly_, something the person who corrected the unknown-target reference bug clearly didn't do and which I didn't check before committing it. So I will fix that up and push in another commit.

Reply:
Fixed. Now I feel dirty. :roll_eyes: